package.json

```diff
@@ -44,6 +44,7 @@
     "preact": "^10.18.1",
     "prettier": "^3.0.3",
     "typescript": "^5.2.2",
-    "url": "^0.11.3"
+    "url": "^0.11.3",
+    "vscode-uri": "^3.0.8"
```

Comment:
You shouldn't need this dependency - `Uri` is already part of the [VS Code API](https://code.visualstudio.com/api/references/vscode-api#Uri). You can just import it from `vscode`